.github/workflows/main.yaml

```diff
@@ -161,6 +161,9 @@ jobs:
         with:
           targets: ${{ needs.initialize.outputs.targets }}
           test_artifacts_key: ${{ env.TEST_ARTIFACTS_KEY }}
+      - name: Upload On Device Test Artifacts
```

Comment:
For on-host tests I put this at the end of the build step. I guess it should be moved to its own action too.

Reply:
Actually, since you named it `upload_test_artifacts` it should probably have both.

I'll take care of that after this merges.

Reply:
ack.